### ISSUE TITLE
Bridges are determined incorrectly

### ISSUE THREAD

**Issue by Mesoptier** [PropaneDragon/Cimtographer#7]
At the moment whether a road is a bridge or not appears to be determined by its elevation. But when using the mod that lets you switch between normal/ground/elevated/bridge construction, this can lead to incorrect classifications. A grounded road with an elevation is rendered as a bridge, instead of as a grounded road which is how it appears in-game.

It would be better to look at the road's elevation type, which should be possible.

**Comment by PropaneDragon** [comment]
Yeah, it does pick based off elevation but also if it's classified as a bridge by the game. I'll see if I can sort this out.
